Resources/Documentation/dv_info.md

```diff
@@ -1,32 +1,126 @@
+
+7. To end playback, click the `x` in the upper left hand corner of the playback window or hit the "esc" key.
+
+
+## Recording
+
+![dv_vrecord_record-instructions](../dv_vrecord_record-instructions.mp4)
+
+1. Select the name of the DV device you want to use from the "Select a DV Device" list.
+
+2. Click the "deck control" button to allow vrecord to control your device.
+
+3. To record, select the recording directory (location where you want to save the file) and enter the recording name, name of the person digitizing this tape, recording time in minutes (if applicable) and any other options you would like applied to your file.
+
+4. Click the "Record" button. The settings menu will close and the Terminal window will display the settings you have entered.
 
+5. When you are ready hit the "Return" button to begin the recording.
+
+6. A mpv window will open displaying the playback of the tape once dvrescue starts the recording. 
+> **_NOTE:_** If the timecode does not start at the very beginning of the tape, the record mode viewer will not open until the timecode is detected (as soon as the counter starts moving on your deck, the viewer window should pop-up). Please see the "Playback" section for an example.
+
+7. When dvrescue encounters an error, the software will automatically rewind and attempt to recapture the problematic frames.
+
+8. To begin another capture on a separate deck, open a new Terminal window and run `vrecord -e`.
+   - Select a different device from the list in the "DV" tab.
+   - Repeat steps 2-7 listed above.  
+
+9. Typically, vrecord will end the capture once the recording ends. If you entered a duration for the capture in "Settings," vrecord will end the capture when it reaches that duration. To end the capture manually, click the `x` in the upper left hand corner of the playback window or hit the "esc" key.
+
+10. The Terminal window will display how the frames were merged for any moments when dvrescue reattempted to recapture problematic frames and how many frames remain with errors in the final recording. 
+
+11. Close the Terminal window.
+
+12. Depending on the settings you selected, a set of logs and the video file should all be saved at the location you selected in the settings.
+
+![dv_vrecord_check-files](../dv_vrecord_check-files.jpg)
+
+
+13. Review your files and logs to ensure they were captured correctly. 
+
+
+## Bitstream Error Concealment
 If vrecord detects that the DV device is concealing bitstream errors, this will be noted in the Terminal.
 ![Alt text](../dv_vrecord_bitstream_concealment.png "Detection of Bitstream Error Concealment")
+
+
+## Known Issues
+
+### No Audio During Playback
```

Comment:
Can get rid of this one - I think I've fixed it in https://github.com/amiaopensource/vrecord/pull/856